infrastructure/global/iam_roles/lambda_roles.tf

```diff
@@ -65,4 +65,16 @@ data "aws_iam_policy_document" "lambda_role" {
     ]
     effect = "Allow"
   }
+
+  # SQS
+  statement {
+    actions = [
+      "sqs:SendMessage",
+      "sqs:DeleteMessage",
+      "sqs:GetQueueAttributes",
+      "sqs:ReceiveMessage",
+    ]
+    resources = ["*"]
+    effect    = "Allow"
+  }
```

Comment:
Having 1 lambda role for all our lambda makes it simpler to manage but it is impossible to apply specific permissions

i.e right now staging can read/delete messages from the production queue which is not ideal.

we should refactor this sooner than later

Reply:
can make an issue for that? :)